Say Their Names/Source/Controller/Home/CollectionViewDataSource/PersonCollectionViewDataSource.swift

```diff
@@ -8,6 +8,8 @@
 
 import UIKit
 
+private let headerIdentifier = "PersonHeaderCell"
```

Comment:
We should not be declaring variables outside of classes. 

Have a look at `CollectionView++.swift` to see how cell reuse identifiers are being used

Reply:
I have since removed this! I missed it.